helm/charts/kratos/values.yaml

```diff
@@ -126,6 +126,8 @@ kratos:
     # - sleep 5;
     #   - kratos
     customArgs: []
+    # -- resource requests and limits for the automigration initcontainer
+    resources: {}
```

Comment:
please add the resources to the job object also :)

Reply:
Done with 2aaa1aafef71fe31b88dac9a84cbe6af602edb89